mappings/net/minecraft/world/gen/heightprovider/HeightProvider.mapping

```diff
@@ -0,0 +1,6 @@
+CLASS net/minecraft/class_6122 net/minecraft/world/gen/heightprovider/HeightProvider
+	FIELD field_31540 CODEC Lcom/mojang/serialization/Codec;
+	METHOD method_35388 getType ()Lnet/minecraft/class_6123;
+	METHOD method_35391 provideHeight (Ljava/util/Random;Lnet/minecraft/class_5868;)I
```

Comment:
Only `get` should suffice

Reply:
but it returns the instance's `HeightProviderType`
`get` would be a bit... obscure

Reply:
oh you mean for `provideHeight`
ok lol